decompile/General/AltMods/OnlineCTR/global.h

```diff
@@ -201,13 +201,14 @@ struct SG_MessageEndRace
 	unsigned char time[3];
 };
 
+#ifdef _WIN32
```

Comment:
We will want to verify the sizes for these structs, we can't guarantee it will be the same when building to other architectures/platforms (like armv7 or android).

So, what goes wrong here?

Reply:
CTR-ModSDK/mods/Windows/OnlineCTR/Network_PC/Server/../../../../../decompile/General/AltMods/OnlineCTR/global.h:204:15: error: expected declaration specifiers or ‘...’ before ‘sizeof’
  204 | static_assert(sizeof(struct SG_Header) == 1);
      |               ^~~~

Reply:
Fixed by @PedroHLC by using a custom `STATIC_ASSERT2`